### ISSUE TITLE
About padding='same'

### ISSUE THREAD

**Issue by 95AliceHong** [BGU-CS-VIL/WTConv#33]
Hello, thank you for your work! 

I noticed that WTConv has requirements for pytorch versions, for example, the padding='same' in
```
        self.base_conv = nn.Conv2d(in_channels, in_channels, kernel_size, padding='same', stride=1, dilation=1,
                                   groups=in_channels, bias=bias)
        self.base_scale = _ScaleModule([1, in_channels, 1, 1])

        self.wavelet_convs = nn.ModuleList(
            [nn.Conv2d(in_channels * 4, in_channels * 4, kernel_size, stride=1, dilation=1,padding='same',
                       groups=in_channels * 4, bias=False) for _ in range(self.wt_levels)]
        )
```
is supported after pytorch 1.9.0, but my current pytorch 1.8.0 does not support it. Is there an alternative solution? Look forward to your reply!

**Comment by shahaffind** [comment]
Hi, please see #29
